JavaDirectory/src/main/java/com/bug/free/invention/api/Models/Job.java

```diff
@@ -0,0 +1,128 @@
+package com.bug.free.invention.api.Models;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Job {
+    @Id int jobID;
+    String jobTitle;
```

Comment:
needs private variables
```suggestion
    private String jobTitle;
```